dashbuilder-client/dashbuilder-common-client/src/main/java/org/dashbuilder/common/client/editor/list/HorizImageListEditorView.java

```diff
@@ -88,7 +93,8 @@ public HorizImageListEditorView() {
         tooltip.setWidget(image);
         tooltip.setContainer("body");
         tooltip.setPlacement(Placement.BOTTOM);
-        tooltip.setShowDelayMs(1);
+        tooltip.setIsAnimated(false);
```

Comment:
the actual fix: disable animation, because it's slowing down hiding of the tooltip on firefox. Also increased showDelayMs, because the original 1ms caused the tooltip to immediately reappear after the call to `hide()` on line 110